specification/confidentialledger/resource-manager/Microsoft.ConfidentialLedger/preview/2022-09-08-preview/managedccf.json

```diff
@@ -0,0 +1,555 @@
+        "operationId": "Mccf_Create",
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ManagedCCFNameParameter"
+          },
+          {
+            "name": "ManagedCCF",
+            "in": "body",
+            "description": "Managed CCF Create Request Body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ManagedCCF"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful and ledger was created; response contains the Managed CCF",
+            "schema": {
+              "$ref": "#/definitions/ManagedCCF"
+            }
+          },
+          "201": {
+            "description": "The Managed CCF resource will be created asynchronously"
+          },
+          "default": {
+            "description": "Error response describing the reason for operation failure. 404 - NotFound (Managed CCF could not be created/updated)",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ManagedCCFCreate": {
+            "$ref": "./examples/Mccf_Create.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "Managed CCF"
+        ],
+        "summary": "Update Managed CCF properties",
+        "description": "Updates properties of Managed CCF",
+        "operationId": "Mccf_Update",
+        "x-ms-long-running-operation": true,
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ManagedCCFNameParameter"
+          },
+          {
+            "name": "ManagedCCF",
+            "in": "body",
+            "description": "Managed CCF request body for Updating MCCF App",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ManagedCCF"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful and ledger was recovered; response contains the Managed CCF",
+            "schema": {
+              "$ref": "#/definitions/ManagedCCF"
```

Comment:
>#/definitions/ManagedCCF [](http://example.com/codeflow?start=23&length=24)

You cannot include resource top-level properties other than tags here. Include only tags and any patchable RP-specific properties in the property bag.

Reply:
@mentat9 Is this something new ? This is the current behavior on the existing resourcetype "ConfidentialLedger".
I've noticed also new errors coming from the existing resourcetype even though has not been changed compared to the GA version

Reply:
Discussed with @sidmore , we can remove it for this scenario